contracts/gauges/LiquidityGaugeV5Collateral.vy

```diff
@@ -733,6 +749,28 @@ def set_killed(_is_killed: bool):
 
     self.is_killed = _is_killed
 
+@external
+def set_whitelist(_custody_contract: address, _is_whitelisted: bool):
+    """
+    @notice Toggle the whitelist status of a custody contract
+    @param _custody_contract Contract that custodies. Used for collateral contract in lending pool
+    @param _is_whitelisted Whether contract is whitelisted
+    """
+    assert msg.sender == self.admin
+
+    self.whitelisted_custody[_custody_contract] = _is_whitelisted
+
+@external
+def decrease_adjusted_balance(_addr: address, _value: uint256):
```

Comment:
We have to add custom logic in the lending contracts to be able to decrease adjusted balance of the borrower whenever they get liquidated, since we do not have access to that information on this contract. 

1. Alice has 10 rETH-THETA-gauge tokens
2. Alice collateralizes her 10 rETH-THETA-gauge tokens on a lending pool to borrow USDC. Her effective balance should remain unchanged and she should continue earning RBN, whereas the receiver `lending pool` should always have adjusted balance of 0.
3. Alice gets liquidated because she is Degen and Bob was the liquidator who covered her loan and received the rETH-THETA-gauge tokens. In the liquidation step, after the transfer of the tokens to Bob, we need to `decrease_adjusted_balance` of Alice because she no longer "holds" those tokens. "Holds" = balance in her wallet + balance used as collateral in lending pool. 

This generalizes to when Alice just deposits into lending pool and immediately withdraws, since the initial transfer does not change the adjustedBalanceOf, but the second transfer adds to her adjusted balance seemingly double-counting. That is why we need to call decrease_adjusted_balance as well in withdraw step of lending pool.